doc/ref_cert/RC2/chapters/chapter02.md

```diff
@@ -172,14 +172,14 @@ are defined as mandatory (e.g. sec.std.001: The Cloud Operator **should**
 comply with Center for Internet Security CIS Controls) else it would have
 required an update of the default kube-bench behavior (all failures and
 warnings are only printed) as integrated in
-[Functest Kubernetes Security](https://git.opnfv.org/functest-kubernetes/tree/docker/security/testcases.yaml?h=stable%2Fkali).
+[Functest Kubernetes Security](https://git.opnfv.org/functest-kubernetes/tree/docker/security/testcases.yaml?h=stable%2Fleguer).
 
-The following software versions are considered to verify Kubernetes v1.19
+The following software versions are considered to verify Kubernetes v1.20
 (latest stable release) selected by CNTT:
 
 | software                | version     |
 |-------------------------|-------------|
-| Functest                | kali        |
+| Functest                | leguer      |
 | kube-hunter             | 0.3.1       |
 | kube-bench              | 0.3.1       |
```

Comment:
should this be v0.5.0 -- added k8s 1.18 as default

Reply:
Please ignore, this wasn't part of my change.